### PR TITLE
Feature/mp 5769/mp 6587 docs issues fix

### PR DESCRIPTION
**Pull request by nPeretyatko**
## PR Description

- Added missing files
- Fixed feature dependecies
- Comments `SomeItem[]` changed to `array<SomeItem>`
- Fixed csv files (first line displayed with error)
- Deleted or moved unrelated verification
- Added tips
- Typos fixed
- Missing translations added
- Missing methods added
- etc.

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.md [hunk 1]
```diff
@@ -900,6 +889,16 @@ class MerchantOmsCommunicationFactory extends SprykerMerchantOmsCommunicationFac
 
 <!--Describe how a developer can check they have completed the step correctly.-->
 
+1. To verify `MerchantReturnPreCreatePlugin` make sure that when you create return for merchant order items, row in `spy_sales_return` that identifies the new return has `spy_sales_return.merchant_reference` field populated.
+2. To verify `MerchantReturnCreateRequestValidatorPlugin` make sure, that you can't create return for items from different merchants.
```
Comment (VadymSachenko) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.md:893]:
```suggestion
2. To verify `MerchantReturnCreateRequestValidatorPlugin`, make sure that you can't create a return for items from different merchants.
```

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.md [hunk 2]
```diff
@@ -900,6 +889,16 @@ class MerchantOmsCommunicationFactory extends SprykerMerchantOmsCommunicationFac
...
+3. To verify `MerchantReturnExpanderPlugin` make sure that you can see merchant order references on return detail page.
```
Comment (VadymSachenko) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.md:894]:
```suggestion
3. To verify `MerchantReturnExpanderPlugin`, make sure that you can see merchant order references on the return details page.
```

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.md [hunk 3]
```diff
@@ -900,6 +889,16 @@ class MerchantOmsCommunicationFactory extends SprykerMerchantOmsCommunicationFac
...
+
+{% endinfo_block %}
+
+{% info_block warningBox "Verification" %}
+
+<!--Describe how a developer can check they have completed the step correctly.-->
+
 Make sure that when you create and process return for merchant order items, it's statuses synced between state machines in the following way:
```
Comment (VadymSachenko) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-return-management-feature-integration.md:902]:
```suggestion
Make sure that when you create and process a return for merchant order items, its statuses are synced between state machines in the following way:
```

File: docs/scos/dev/feature-integration-guides/202009.0/availability-notification-feature-integration.md [hunk 4]
```diff
@@ -323,7 +322,7 @@ console data:import glossary
 ```
 
 {% info_block warningBox "Verification" %}
-Make sure that, in the database, the configured data is added to the `spy_glossary` table.
+Make sure that, in the database, the configured data is added to the `spy_glossary_key` and `spy_glossary_translation` tables.
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202009.0/availability-notification-feature-integration.md:325]:
```suggestion
Make sure that in the database, the configured data is added to the `spy_glossary_key` and `spy_glossary_translation` tables.
```

File: docs/scos/dev/feature-integration-guides/202009.0/multiple-carts-feature-integration.md [hunk 5]
```diff
@@ -79,7 +78,7 @@ console data:import glossary
 ```
 
 {% info_block warningBox "Verification" %}
-Make sure that  the configured data has been added to the `spy_glossary` table in the database.
+Make sure that  the configured data has been added to the `spy_glossary_key` and `spy_glossary_translation` tables in the database.
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202009.0/multiple-carts-feature-integration.md:81]:
```suggestion
Make sure that the configured data has been added to the `spy_glossary_key` and `spy_glossary_translation` tables in the database.
```

File: docs/scos/dev/feature-integration-guides/202108.0/checkout-feature-integration.md [hunk 6]
```diff
@@ -596,7 +595,7 @@ console data:import glossary
 
 {% info_block warningBox "Verification" %}
 
-Ensure that, in the database, the configured data has been added to the `spy_glossary` table.
+Ensure that, in the database, the configured data has been added to the `spy_glossary_key` and `spy_glossary_translation` tables.
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202108.0/checkout-feature-integration.md:598]:
```suggestion
Ensure that in the database, the configured data has been added to the `spy_glossary_key` and `spy_glossary_translation` tables.
```

File: docs/scos/dev/feature-integration-guides/202108.0/gift-cards-feature-integration.md [hunk 7]
```diff
@@ -1239,7 +1238,7 @@ console data:import:glossary
 
 {% info_block warningBox "Verification" %}
 
-Make sure that, in the database, the configured data has been added to the `spy_glossary` table.
+Make sure that, in the database, the configured data has been added to the `spy_glossary_key` and `spy_glossary_translation` tables.
```
Comment (VadymSachenko) [inline docs/scos/dev/feature-integration-guides/202108.0/gift-cards-feature-integration.md:1241]:
```suggestion
Make sure that in the database, the configured data has been added to the `spy_glossary_key` and `spy_glossary_translation` tables.
```

**Review by VadymSachenko — APPROVED** [review]

**Comment by asmarovydlo** [comment]
This PR was reverted because had no approval.
Will be merged again after approval as part of the https://github.com/spryker/spryker-docs/pull/908 PR.

**Pull request merged**